pic_email.py

```diff
@@ -10,7 +10,7 @@
 
 def write_to_pic(name, email, token):  # 执行完这个方法后生成一个 result.png 图片 可加入email参数
     number = get_number(email)  # 用这个方法获取到编号
-    date_str = datetime.datetime.strftime(datetime.datetime.now(),'%Y年%m月%d日') #获取日期
+    date_str = datetime.datetime.strftime(datetime.datetime.now(), u'%Y{0}%m{1}%d{2}').format(*'年月日')  # 获取日期
```

Comment:
Do not use u'xxx'. This is Python 2 usage.

Reply:
Oh, yes! I forget to delete it, it's useless in Python3.